youtube_dl/extractor/imgur.py

```diff
@@ -100,7 +109,7 @@ def _real_extract(self, url):
 
 
 class ImgurAlbumIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:i\.)?imgur\.com/gallery/(?P<id>[a-zA-Z0-9]+)'
+    _VALID_URL = r'https?://(?:i\.)?imgur\.com/(gallery/)?(?P<id>[a-zA-Z0-9]{5})(?![a-zA-Z0-9])'
```

Comment:
http://imgur.com/Q95ko should be handled, hence the change `gallery/` -> `(gallery/)?`
Also in case of albums, the id is of length 5, so incorporated that.